examples/chat-online-status/javascript/src/script.ts

```diff
@@ -1,10 +1,11 @@
 import * as Ably from 'ably';
 import { ChatClient, PresenceEvent, PresenceMember, AllFeaturesEnabled } from '@ably/chat';
-import { faker } from '@faker-js/faker';
+import minifaker from 'minifaker';
```

Comment:
`minifaker` was brought in purely because `faker` blows up sandpack for reasons long and arduous